src/lib/eina/eina_value.h

```diff
@@ -339,6 +339,15 @@ union _Eina_Value_Union
    uint64_t      _guarantee; /**< guarantees 8-byte alignment */
 };
 
+
+/**
+ * @var EINA_VALUE_TYPE_ERROR
+ * manages Eina_Error values.
```

Comment:
give bit more context here. Say:

This value will hold an `Eina_Error` number, which can be converted to string to get its message or call `eina_error_msg_get()` on the number to convert yourself.